tests/phpunit/helpers/class-wc-helper-stripe-api.php

```diff
@@ -14,16 +14,88 @@
 class WC_Helper_Stripe_Api {
 
```

Comment:
The rest of the changes below are unit tests. Because `WC_Stripe_Api` is a static class, mocking the returned values is finicky.